example/src/main/kotlin/com.expedia.graphql.sample/query/UnionQuery.kt

```diff
@@ -0,0 +1,17 @@
+package com.expedia.graphql.sample.query
+
+import org.springframework.stereotype.Component
+
+@Component
+class QueryWithUnion: Query {
```

Comment:
do we want to move this to `PolymorphicQuery` instead?

Reply:
👍